liveness/src/main/java/com/amplifyframework/ui/liveness/state/LivenessState.kt

```diff
@@ -47,15 +46,14 @@ internal data class LivenessState(
     val context: Context,
     val disableStartView: Boolean,
     val onCaptureReady: () -> Unit,
-    val onFaceDistanceCheckPassed: () -> Unit,
     val onSessionError: (FaceLivenessDetectionException, Boolean) -> Unit,
     val onFinalEventsSent: () -> Unit,
 ) {
     var videoViewportSize: VideoViewportSize? by mutableStateOf(null)
     var livenessCheckState = mutableStateOf<LivenessCheckState>(
```

Comment:
Should use property delegate
```suggestion
    var livenessCheckState by mutableStateOf<LivenessCheckState>(
```